contracts/modules/SuperMinterV1_1.sol

```diff
@@ -303,20 +322,18 @@ contract SuperMinterV1_1 is ISuperMinterV1_1, EIP712 {
     /**
      * @inheritdoc ISuperMinterV1_1
      */
-    function mintTo(MintTo calldata p) public payable {
+    function mintTo(MintTo calldata p) public payable returns (uint256 fromTokenId) {
         MintData storage d = _getMintData(LibOps.packId(p.edition, p.tier, p.scheduleNum));
 
         /* ------------------- CHECKS AND UPDATES ------------------- */
 
-        // Check if the mint is open.
-        if (LibOps.or(block.timestamp < d.startTime, block.timestamp > d.endTime))
-            revert MintNotOpen(block.timestamp, d.startTime, d.endTime);
-        if (_isPaused(d)) revert MintPaused(); // Check if the mint is not paused.
+        _requireMintOpen(d);
 
         // Perform the sub workflows depending on the mint mode.
         uint8 mode = d.mode;
         if (mode == VERIFY_MERKLE) _verifyMerkle(d, p);
         else if (mode == VERIFY_SIGNATURE) _verifyAndClaimSignature(d, p);
+        else if (mode == PLATFORM_AIRDROP) revert InvalidMode();
```

Comment:
This is important, the main risk creating a new state involves running older functions in unexpected states. This keeps `mintTo` from being called for platform airdrops. This comment is to note down where the expected mitigations are, no response needed.